app/scripts/metamask-controller.js

```diff
@@ -3323,6 +3323,7 @@ export default class MetamaskController extends EventEmitter {
             transactionOptions,
             waitForSubmit: false,
           }),
+          accountsController.listAccounts(),
```

Comment:
The intent of the above `AddTransactionRequest` argument is to contain the many arguments needed to add the transaction.

Can we move this into that for consistency?

Reply:
PR is updated.